bsd/models.py

```diff
@@ -52,8 +55,13 @@ def from_json(self, data):
             )
             venue_state_or_territory = data["venue_state_cd"]
 
-        """Fix duration format"""
-        duration_count = int(duration) if duration != '' else 0
+        """Set duration to a positive integer if possible"""
+        """TODO: support all day events (-1)?"""
```

Comment:
If an all-day event already exists in BSD will that break anything here?

Reply:
If all day event exists in BSD, app will set duration to none here, and then Update form will require a new value for duration field. If we want to support all day events we need to do some more work here. Apparently hydra did not support it.

Reply:
Okay that is fine, thanks for explaining.